about/faq.rst

```diff
@@ -207,6 +207,48 @@ In the long term, we may develop a Direct3D 12 renderer for Godot (mainly for
 the Xbox's purposes), but Vulkan and OpenGL will remain the default rendering
 backends on all platforms, including Windows.
 
+Why does Godot aim to keep its core feature set small?
+------------------------------------------------------
+
+Godot intentionally does not include features that can be implemented by add-ons
+unless they are used very often. A few examples of this would be artificial
+intelligence or terrain systems.
```

Comment:
I wouldn't mention terrain here. The thinking around terrain has shifted and I think the only thing stopping it from being added is the amount of work necessary to do it properly/